### PR TITLE
global: bump invenio packages versions

### LINKED ISSUE #749: installation issues
Hi,

Getting errors when trying to install using the scripts provided. It seems there is a conflict.

1. First, get this error:
ERROR: invenio-stats 1.0.0a15 has requirement kombu<4.4,>=4.2.0, but you'll have kombu 4.6.7 which is incompatible.
ERROR: invenio-celery 1.1.1 has requirement celery<4.3.0,>=4.2.1, but you'll have celery 4.4.0 which is incompatible.

2. Then upon uninstalling 4.4 and installing 4.2.1, get this error:
ERROR: flask-celeryext 0.3.3 has requirement celery>=4.3; python_version == "3.7", but you'll have celery 4.2.1 which is incompatible.

Plz address.
Thanks,
mj

### PR DESCRIPTION
**Pull request by kpsherva**
* updates invenio-celery, invenio-stats
* removes Flask-celeryext
* fixes ui components

closes #749

### CODE REVIEW

File: ui/src/authentication/components/AuthenticationGuard/AuthenticationGuard.js [hunk 1]
```diff
@@ -34,11 +35,12 @@ export default class AuthenticationGuard extends Component {
     }
 
     if (!isAnonymous && !authenticationService.hasRoles(user, roles)) {
-      this.props.sendErrorNotification(
```
Comment (kpsherva) [inline ui/src/authentication/components/AuthenticationGuard/AuthenticationGuard.js:37]: this was creating notifications for "hidden" components

File: ui/src/authentication/services/SessionManager.js [hunk 2]
```diff
@@ -8,6 +8,10 @@ class SessionManager {
     return this.user === null;
   }
 
+  isAuthenticated() {
```
Comment (kpsherva) [inline ui/src/authentication/services/SessionManager.js:11]: fixes some leftovers after SPA login refactoring

File: ui/src/semantic-ui/site/collections/menu.overrides [hunk 3]
```diff
@@ -2,6 +2,20 @@
          Site Overrides
 *******************************/
 
+.ui.inverted.menu {
```
Comment (kpsherva) [inline ui/src/semantic-ui/site/collections/menu.overrides:5]: fixes the Login button broken style on the home page

File: ui/src/authentication/services/SessionManager.js (hunk 2 above)
Comment (ntarocco) [reply ui/src/authentication/services/SessionManager.js:11]: This was already done in @topless PR, right?

**Review by ntarocco — APPROVED** [review]

**Pull request merged**